systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -24,35 +24,41 @@
 @ClusterOperator
 public class SecurityST extends AbstractST {
 
-    public static final String NAMESPACE = "security-cluster-test";
+    public static final String NAMESPACE = "myproject";
```

Comment:
Is this change intentional? Are we now running the tests in the `myproject` namespace? (just curious)

Reply:
Ah, damn. Thanks I will fix this.